packages/webview_flutter/webview_flutter_platform_interface/lib/src/v4/types/web_resource_error.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+import 'web_resource_error_type.dart';
+
+/// Error returned in `WebView.onWebResourceError` when a web resource loading error has occurred.
+@sealed
```

Comment:
Im not sure if we want this sealed since it prevents `extends`. A platform may want to extend this to add additional parameters. For example, Android has more parameters in [onRecievedError](https://developer.android.com/reference/android/webkit/WebViewClient#onReceivedError(android.webkit.WebView,%20android.webkit.WebResourceRequest,%20android.webkit.WebResourceError))

Reply:
Removed the `@sealed` attribute and extended the class with the `PlatformInterface`.